src/openforms/js/components/admin/forms/MappingArrayInput.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import {ArgsTable, Canvas, Meta, Story} from '@storybook/addon-docs';
+
+import ArrayInput from './ArrayInput';
+import MappingArrayInput from './MappingArrayInput';
+
+<Meta title="Admin/Custom/MappingArrayInput" component={MappingArrayInput} />
```

Comment:
there are a bunch of warnings in the console for these stories, please resolve those